include/config/battle.h

```diff
@@ -267,5 +267,7 @@
 #define B_NEW_MORNING_SUN_STAR_PARTICLE TRUE    // If set to TRUE, it updates Morning Sun's star particles.
 #define B_NEW_IMPACT_PALETTE            TRUE    // If set to TRUE, it updates the basic 'hit' palette.
 #define B_NEW_SURF_PARTICLE_PALETTE     TRUE    // If set to TRUE, it updates Surf's wave palette.
+#define B_OPPONENT_THROW_BALLS          GEN_LATEST // In GEN_6+, opposing Trainers throw PokéBalls into battle instead of just dropping them.
+#define B_THROW_BALLS_SOUND             GEN_LATEST // In GEN_5+, Trainers PokéBalls make a sound when thrown to send out a Pokémon.
```

Comment:
I think it would make sense to connect `B_OPPONENT_THROW_BALLS` to `B_OPPONENT_THROW_BALLS` but not vice versa.
so
```
if  B_OPPONENT_THROW_BALLS is set to false, turn off `B_OPPONENT_THROW_BALLS`
```
but you would still be able to turn off sound individually. 

To be completely honest I would combine both but I understand why you've made it this way.

Reply:
To be clear , you are saying

> Throw sound doesn't work unless throw balls is on

If you are saying that, are you also implicitly saying

> There is no sound effect for the player throwing their ball unless the enemy is throwing theirs 

Or do you think I should split it up into
- player throw sound 
- enemy throw
- enemy throw sound (only possible when enemy throw is on)

Reply:
I'm saying that there is a sound effect for the enemy if `B_OPPONENT_THROW_BALLS` is off which is a bit weird.

Reply:
I understand that, but executing your suggestion has side effects that I'm trying to confirm are intended or an oversight.

Reply:
So the player should still have it's sound effect. To be honest this is something that would be more easily solved if we didn't config that at all. Maybe this is worth asking in the senate instead of writing questionable code.

Reply:
I don't think there is anywhere near important enough to disturb the whole Senate, and I mostly agree with you anyways.

If somebody doesn't like it later, they're welcome to remove the config or merge them back into one - it wasn't hard or a lot of work.

https://github.com/rh-hideout/pokeemerald-expansion/pull/4953/commits/e0ca4216422eea32754478da82fdc8831b966ebb